geometric_kernels/spaces/hyperbolic.py

```diff
@@ -8,11 +8,18 @@
 import lab as B
 from opt_einsum import contract as einsum
 
-from geometric_kernels.lab_extras import cosh, from_numpy, logspace, sinh, trapz
-from geometric_kernels.spaces import Space
-
-
-class Hyperbolic(Space, gs.geometry.hyperboloid.Hyperboloid):
+from geometric_kernels.lab_extras import (
+    cosh,
+    dtype_double,
+    from_numpy,
+    logspace,
+    sinh,
+    trapz,
+)
+from geometric_kernels.spaces import NoncompactSymmetricSpace
+
+
+class Hyperbolic(NoncompactSymmetricSpace, gs.geometry.hyperboloid.Hyperboloid):
```

Comment:
Would it be hard to implement this thing, similar to geomstats, for multiple models of the hyperbolic space?

Reply:
Not hard per se. Unwieldy maybe, I imagine. However, I don't see much point in this, and this seems a bit outside of the scope of this PR?

Reply:
Agree, out of scope